### PR TITLE
Rename stsServer to authority

### PR DESCRIPTION
**Pull request by damienbod**

### CODE REVIEW

File: docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md [hunk 1]
```diff
@@ -595,3 +595,25 @@ const appRoutes: Routes = [
   { path: 'unauthorized', component: UnauthorizedComponent },
 ];
 ```
+
+### `stsServer` --> `authority`
+
+Renaming the stsServer to authority. The authority issues tokens. Sometimes this is called issuer, secure token service, auth server, identity provider. The authority is sometimes known as the issuer.
```
Comment (FabianGosebrink) [inline docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md:601]: Please add backticks around property names

File: docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md [hunk 2]
```diff
@@ -595,3 +595,25 @@ const appRoutes: Routes = [
...
+
+### `stsServer` --> `authority`
+
+Renaming the stsServer to authority. The authority issues tokens. Sometimes this is called issuer, secure token service, auth server, identity provider. The authority is sometimes known as the issuer.
+
+Old:
+
+```ts
+export interface OpenIdConfiguration {
+  autoUserinfo?: boolean;
```
Comment (FabianGosebrink) [inline docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md:607]: Not `autoUserinfo` here ;)

File: docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md (hunk 1 above)
Comment (damienbod) [reply docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md]: fixed

File: docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md (hunk 2 above)
Comment (damienbod) [reply docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md]: using stsServer to authority

File: projects/angular-auth-oidc-client/src/lib/config/validation/rules/ensure-authority.rule.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { OpenIdConfiguration } from '../../openid-configuration';
 import { POSITIVE_VALIDATION_RESULT, RuleValidationResult } from '../rule';
 
-export const ensureStsServer = (passedConfig: OpenIdConfiguration): RuleValidationResult => {
-  if (!passedConfig.stsServer) {
+export const ensureAuthority = (passedConfig: OpenIdConfiguration): RuleValidationResult => {
+  if (!passedConfig.authority) {
     return {
       result: false,
       messages: ['The STS URL MUST be provided in the configuration!'],
```
Comment (FabianGosebrink) [inline projects/angular-auth-oidc-client/src/lib/config/validation/rules/ensure-authority.rule.ts:8]: Can you fix this message as well?

Comment (damienbod) [reply projects/angular-auth-oidc-client/src/lib/config/validation/rules/ensure-authority.rule.ts]: fixed

**Pull request merged**